src/theme.ts

```diff
@@ -43,6 +43,19 @@ export const theme = {
     default: '0 2px 6px 0 rgba(0, 0, 0, 0.2);',
   },
 
+  layout: {
+    default: `
+    display: flex;
+    width: 100%;
+    flex-direction: column;
+    padding: 32px 52px 52px;
+    box-sizing: border-box;
+    @media screen and (max-width: 768px){
+      padding: 1rem;
+    }
```

Comment:
Setting display mode and flex-direction is probably a bad idea
```suggestion
    width: 100%;
    padding: 32px 52px 52px;
    box-sizing: border-box;
    @media screen and (max-width: 768px){
      padding: 1rem;
    }
```